### PR TITLE
cleanup subscriptions page interface

### PR DESCRIPTION
**Pull request by asood123**
UI tweaks to make this page easier to use: https://github.com/OpenCollective/OpenCollective/issues/224

### CODE REVIEW

File: frontend/src/components/SubscriptionItem.js [hunk 1]
```diff
@@ -25,19 +25,19 @@ export default class SubscriptionItem extends Component {
 
     const formattedAmount = formatCurrency(amount*100, currency, {compact: true});
     const formattedInterval = `${interval[0].toUpperCase()}${interval.substr(1)}ly`;
-    const formattedCreatedAt = isActive ? `${i18n.getString('since')} ${moment(createdAt).format('MMM, YYYY')}`: 'Inactive';
+    const formattedCreatedAt = isActive ? `${i18n.getString('since')} ${moment(createdAt).format('MMM, YYYY')}`: 'Cancelled';
```
Comment (xdamman) [inline frontend/src/components/SubscriptionItem.js]: you should use `i18n.moment` to make sure it uses the right locale.

File: frontend/src/css/components/SubscriptionItem.css [hunk 2]
```diff
@@ -21,7 +21,7 @@
       font-weight: 400;
       font-style: italic;
       text-align: right;
-      color: #b3b3b3;
+      color: #dd2663;
```
Comment (xdamman) [inline frontend/src/css/components/SubscriptionItem.css]: better to use variables for colors for consistency.

File: frontend/src/css/components/SubscriptionItem.css [hunk 3]
```diff
@@ -37,11 +37,13 @@
       display: inline-block;
       width: 110px;
       height: 110px;
-      background-color: #ffffff;
+      background-color: ##DDE1E4;
```
Comment (xdamman) [inline frontend/src/css/components/SubscriptionItem.css]: same

**Comment by asood123** [comment]
Modified based on comments. Merging.

**Pull request merged**